### ISSUE TITLE
Windows: error auto-installing `dart-sass`

### ISSUE THREAD

**Issue by akesson** [akesson/cargo-leptos#21]
On windows, when running `cargo leptos watch` without sass installed:
```
Leptos rebuild stopped due to error: at `src\main.rs@147:33`

Caused by:
    0: at `src\main.rs@155:30`
    1: compile sass/scss: style/main.scss at `src\run\sass.rs@24:14`
    2: Try manually installing sass: https://sass-lang.com/install at `src\run\sass.rs@45:26`
    3: Unable to download sass for windows x86_64 due to: failed to download from https://github.com/sass/dart-sass/releases/download/1.56.1/dart-sass-1.56.1-windows-x64.zip
```

The error is thrown by the [binary-install](https://docs.rs/binary-install/0.0.2/binary_install/) crate.

Preliminary investigation shows that the cache dir `AppData/Local/.cargo-leptos` is created, but nothing is extracted to it.

Any help solving this would be appreciated as I don't have Windows myself.

**Comment by akesson** [comment]
Windows 11 virtual machine [download](https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/)

**Comment by AshSimmonds** [comment]
When testing this (on Discord recently) I wasn't able to tell why the download itself wasn't happening, and it may just be my machine which might have certain ports blocked or such, not sure as it's fairly well locked down. So this may well still work, just need more Windows folk to test on.

**Comment by akesson** [comment]
Fixed by #24
